tests/events/events.go

```diff
@@ -0,0 +1,107 @@
+package events
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests/util"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// ExpectNoEvent is safe to use in parallel as long as you are asserting namespaced object that is not shared between tests
+func ExpectNoEvent(object any, eventType, reason string) {
+	By("Expecting for an event to be not triggered")
+	expectEvent(object, eventType, reason, BeEmpty())
+}
+
+// ExpectEvent is safe to use in parallel as long as you are asserting namespaced object that is not shared between tests
+func ExpectEvent(object any, eventType, reason string) {
```

Comment:
Thanks, I had to miss this case. I am not convinced we need to assert the message so I just proposed using the new package.

Reply:
Currently, it can be enough. At least until we do not want to distinguish between 
`libvirt.DOMAIN_DISK_ERROR_NO_SPACE` and `libvirt.DOMAIN_DISK_ERROR_UNSPEC`.